NETDefault/RequestHandlerNETCore31/HelperClasses/HttpRequestExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace RequestHandlerNETCore31.HelperClasses
+{
+    internal static class HttpRequestExtensions
+    {
+        public static async Task<string> ReadAsStringAsync(this HttpRequest request)
+        {
+            request.EnableBuffering();
+            string result = null;
+            using (var reader = new StreamReader(request.Body, Encoding.UTF8, detectEncodingFromByteOrderMarks: true, 1024, leaveOpen: true))
+            {
+                result = await reader.ReadToEndAsync();
+            }
+
+            request.Body.Seek(0L, SeekOrigin.Begin);
+            return result;
+        }
+    }
+}

```

Comment:
This code is copied from HttpRequestExtensions in order to lose the dependency on _Microsoft.Azure.WebJobs.Extensions.Http_.

Reply:
Why not keep the `Microsoft.Azure.WebJobs.Extensions.Http` dependency because this handler is only used in conjunction with Azure Functions?

Reply:
Because for a generic example I did not want to take that dependency. I could add a comment there that when used with Azure Functions, one can take the dependency, for the package will be used for other features as well?